BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Enums/BugsnagSpanCatagory.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace BugsnagUnityPerformance
+{
+    public enum BugsnagSpanCatagory
```

Comment:
`catagory` &rarr; `category`

Reply:
decided to remove this enum and stick with span name comparison for now.